### PR TITLE
feat: playground duckdb

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

**Comment by max-sixty** [comment]
As discussed on discord, this is huge!

**Comment by snth** [comment]
Totally agree, this is amazing!

**Comment by snth** [comment]
Minor niggle, the dates in employees.prql and invoices.prql are difficult to interpret as they are currently displayed:

image

If I had to guess, I would say you probably need some conversion around Output.js:63 depending on the column type.

File: playground/src/output/Output.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      const arrow = this.props.content.arrow;
+
+      const header = arrow.schema.fields.map((f, index) => {
+        return <th key={index}>{f.name}</th>;
+      });
+
+      const data = arrow.toArray().map((x) => [...x]);
+      const rows = data.map((x, index) => {
+        const cells = x.map(([_name, value], index) => (
+          <td key={index}>{"" + value}</td>
```
Comment (aljazerzen) [inline playground/src/output/Output.js:63]: @snth you exactly right. Here I'd have to do a `new Date(value)` when type is timestamp/date.

That could be determined from looking into `arrow.schema.fields[index].type`.

I probably won't get around to doing this, but I'm leaving a comment here for anyone who might want to.

**Pull request merged**